packages/amplify/amplify_flutter/lib/src/hybrid_impl.dart

```diff
@@ -24,6 +24,8 @@ import 'package:amplify_flutter/src/amplify_impl.dart';
 class AmplifyHybridImpl extends AmplifyClassImpl {
   /// {@macro amplify_flutter.amplify_hybrid_impl}
   AmplifyHybridImpl() : super.protected();
+  final AmplifyAuthProviderRepository _authProviderRepo =
```

Comment:
Honestly, may just want to make this `@protected` instead of private and passing it into `configure`. If we ever make it public, which I think we should, then all the `configure` methods need to be updated again which is potentially a breaking change.